src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/MapVisualizerPanel.tsx

```diff
@@ -30,6 +30,7 @@ const MapVisualizerPanel: React.FC = () => {
       if (wasExistingLayer) {
         // remove the existing layer from the app state
         dispatch(GeoActions.removeForestLayer(mapLayerKey))
+        dispatch(GeoActions.resetOpacity(mapLayerKey))
```

Comment:
Dispatching double action in this case could be avoided. you can simply set the opacity to 0. in removeForestLayer reducer you can set state.forestOptions.opacity[key] = 0

Reply:
@minotogna Good catch! Fixed now.

Reply:
Thanks @joonasnuutinen . Is this pr good to be merged or is there a reason why it’s still in draft ?

Reply:
@minotogna If #1902 had been merged before this one, there would've been merge conflicts. However, we can merge this first and fix the conflicts in #1902 instead.

Reply:
ah ok, waiting for @eraviolo review on the other one .